tests/src/Opulence/Environments/EnvironmentTest.php

```diff
@@ -37,13 +37,7 @@ public function testGettingVariable()
      */
     public function testIsRunningInConsole()
     {
-        $isRunningInConsole = Environment::isRunningInConsole();
-
-        if (php_sapi_name() == "cli") {
-            $this->assertTrue($isRunningInConsole);
-        } else {
-            $this->assertFalse($isRunningInConsole);
-        }
+        $this->assertEquals(Environment::isRunningInConsole(), php_sapi_name() === "cli");
```

Comment:
because test value and `Environment::isRunningInConsole()` implementation - same thing

if we inline `Environment::isRunningInConsole()` in this test:
` $this->assertEquals(php_sapi_name() === "cli", php_sapi_name() === "cli"); `
this assert can`t be false